CMakeLists.txt

```
@@ -78,6 +78,7 @@ option(ENABLE_DEVMODE      "Compile vesion for developers" off)
 option(ENABLE_UI           "Compile UI (turn off to compile CLI-only version)" on)
 option(FULLSCREEN_PLATFORM "Enable fullscreen by default" off)
 option(ENABLE_CLANG_TIDY   "Enable static analysis" off)
+option(ENABLE_CCACHE       "Use CCache to improve recompilation speed (optional)" ON)
```

I'm not sure if this option should be on or off by default, but I think that if it's on will not make no harm
```suggestion
option(ENABLE_CCACHE       "Use CCache to improve recompilation speed (optional)" on)
```